src/bot/commands/whois.js

```diff
@@ -67,7 +67,7 @@ async function bansLine (userID, guildID, db) {
 		}).toArray();
 
 		return `__Bans: **${numResults}**__${
-			results.map(ban => `\n- ${escape(formatDate(ban.date))}${ban.note ? `: ${escape(ban.note)}` : ''} - <@${ban.modID}>`).join('')
+			results.map(ban => `\n- ${escape(formatDate(ban.date))}${ban.note && ban.expirationDate ? `: **\`${Math.ceil((ban.expirationDate - ban.date) / (1000 * 60 * 60 * 24))}d\` |** ${escape(ban.note)}` : ''} - <@${ban.modID}>`).join('')
```

Comment:
> `ban.note && ban.expirationDate ? ...`

This logic seems off. Permabans have no expiration date (`ban.expirationDate` would report `undefined`), so it looks like this would cause the note on permabans not to be displayed. The expiration date should probably be its own thing.

```suggestion
			results.map(ban => `\n- ${escape(formatDate(ban.date))}${ban.expirationDate ? ` **\`${Math.ceil((ban.expirationDate - ban.date) / (1000 * 60 * 60 * 24))}d\` |**` : '`Permanent`'}${ban.note ? `: ${escape(ban.note)}` : ''} - <@${ban.modID}>`).join('')
```

...god this is a mess. Maybe it's time to just break this formatting out into its own function so we can clean it up a bit.

This suggestion may also not be the exact format we want, so feel free to tweak, but please do test with displaying both temporary and permanent bans to make sure it looks okay